python/py-virtualfish/Portfile

```diff
@@ -28,6 +28,7 @@ python.pep517_backend   poetry
 
 if {${name} ne ${subport}} {
     depends_lib-append  port:py${python.version}-virtualenv \
+                        port:py${python.version}-psutil \
```

Comment:
there are more dependencies listed than those in upstream's pyproject.toml file. Can you please verify and if there are indeed needed add them here explicitly?

Reply:
fixed